database/src/migrations/20240304143200_event_schema.ts

```diff
@@ -0,0 +1,90 @@
+import { Knex } from 'knex';
+
+/**
+ * Create new tables:
+ * - observation_subcount
+ * - subcount_critter
+ * - subcount_event
+ *
+ * Create new function/trigger:
+ * - tr_observation_subcount_count
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`--sql
+    ----------------------------------------------------------------------------------------
+    -- DROP subcount_event table
+    ----------------------------------------------------------------------------------------
+    DROP VIEW IF EXISTS biohub_dapi_v1.subcount_event;
+    DROP TABLE IF EXISTS subcount_event;
+
+    ----------------------------------------------------------------------------------------
+    -- Create measurement tables
+    ----------------------------------------------------------------------------------------
+    CREATE TABLE observation_subcount_quantitative_measurement (
+      observation_subcount_id INTEGER NOT NULL,
+      measurement_quantitative_id UUID NOT NULL,
```

Comment:
Forgot to include this before: to date, we've been prefixing external ids with the name of the system, so like `critterbase_measurement_quantitative_id`